lib/widgets/content.dart

```diff
@@ -1351,8 +1360,5 @@ InlineSpan _errorUnimplemented(UnimplementedNode node) {
   }
 }
 
-const errorStyle = TextStyle(
```

Comment:
content [nfc]: Move codeStyle to ContentTheme

Do you mean s/codeStyle/errorStyle/?